sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1449,6 +1449,13 @@ object SQLConf {
     .intConf
     .checkValues((1 to 9).toSet + Deflater.DEFAULT_COMPRESSION)
     .createWithDefault(Deflater.DEFAULT_COMPRESSION)
+
+  val COMPUTE_STATS_LIST_FILES_IN_PARALLEL =
+    buildConf("spark.sql.execution.computeStatsListFilesInParallel")
+      .internal()
+      .doc("If True, File listing for compute statistics is done in parallel.")
```

Comment:
When true, SQL commands use parallel file listing, as opposed to single thread listing. This usually speeds up commands that need to list many directories.